src/main/java/duke/Storage.java

```diff
@@ -50,6 +61,11 @@ public TaskList load() throws IOException, DukeException {
         return tasks;
     }
 
+    /**
+     * Updates the records according to the new list of tasks.
+     * @param tasks the new list of tasks
+     * @throws IOException if an I/O error occurred
+     */
     public void update(TaskList tasks) throws IOException {
         File records = new File(filePath + "/data/records.txt");
```

Comment:
A slight tendency to misname variables and use plural for singular objects, which can be misleading and is discouraged in the textbook. However, still very neat thus far :).